src/components/AddPeopleAndRoles/ListPeopleAndRoles.vue

```diff
@@ -0,0 +1,253 @@
+            </v-col>
+          </div>
+          <div v-else>
+             <v-icon color="rgb(232, 156, 48)" small>mdi-alert</v-icon>
+             <span class="warningText">Add Role</span>
+          </div>
+        </v-col>
+
+        <!-- Actions Column -->
+        <v-col sm="1" v-if="!isSummary">
+          <div class="actions">
+            <span class="edit-action">
+              <v-btn small text color="primary"
+                :id="'person-' + person.id + '-change-btn'"
+                @click="emitPersonInfo(index)"
+              >
+                <v-icon small>mdi-pencil</v-icon>
+                <span>Edit</span>
+              </v-btn>
+            </span>
+
+            <!-- more actions menu -->
+            <span>
+              <v-menu offset-y>
+                <template v-slot:activator="{ on }">
+                  <v-btn text small color="primary" class="actions__more-actions__btn" v-on="on">
+                    <v-icon>mdi-menu-down</v-icon>
+                  </v-btn>
+                </template>
+                <v-list class="actions__more-actions">
+                  <v-list-item @click="confirmRemove(index)">
+                    <v-list-item-title><v-icon>mdi-delete</v-icon>Remove</v-list-item-title>
+                  </v-list-item>
+                </v-list>
+              </v-menu>
+            </span>
+          </div>
+        </v-col>
+      </v-row>
+    </div>
+  </v-card>
+</template>
+
+<script lang="ts">
+// Libraries
+import { Component, Prop, Mixins, Emit } from 'vue-property-decorator'
+
+// Components
+import BaseAddress from 'sbc-common-components/src/components/BaseAddress.vue'
+
+// Dialogs
+import { ConfirmRemoveDialog } from '@/components/dialogs'
+
+// Mixins
+import CommonMixin from '@/mixins/common-mixin'
+
+// Interfaces
+import { OrgPersonIF } from '@/interfaces'
+
+@Component({
+  components: {
+    BaseAddress,
+    ConfirmRemoveDialog
+  }
+})
+export default class ListPeopleAndRoles extends Mixins(CommonMixin) {
+  // Store Properties
+  @Prop({ default: '' })
+  private personList: Array<OrgPersonIF>
+
+  @Prop({ default: false })
+  private showErrorSummary: boolean
+
+  @Prop({ default: false })
+  private isSummary: boolean
+
+  // Local Properties
+  readonly tableHeaders: Array<string> = ['Name', 'Mailing Address', 'Delivery Address', 'Roles']
+  private dialog: boolean = false
+  private personId: number
+
+  /**
+   * Determine if Corporation/Firm or Person.
+   * @param filing The filing body which contains the name/title.
+   * @returns The appropriate Corporation or Person name.
+   */
+  private formatName (filing: any): string {
+    return filing.orgName ? filing.orgName
+      : `${filing.firstName} ${filing.middleName || ''} ${filing.lastName}`
+  }
+
+  /**
+   * Identify and confirm the removal of a person/org through dialog.
+   * @param index The active index which is subject to removal.
+   */
+  confirmRemove (index: number): void {
```

Comment:
I should have made this private.

Reply:
I read somewhere that "private" didn't really work until the latest version of TS.

Reply:
oh, good to know! :P